sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -527,7 +527,8 @@ def test_binomial():
     assert unchanged(binomial, nt, Rational(5, 2))
     assert binomial(kn - 3, kn - 1, evaluate=False).rewrite(gamma) == 0
     b = binomial(nt, Rational(5, 2))
-    assert b.rewrite(gamma) == b  # nt could be 3/2
```

Comment:
I'll let you reverse the changes for the reasons described in the previous comments.